portal/server/controllers/utilisation-report-service/utilisation-report-upload/utilisation-report-cell-validators/generate-payment-exchange-rate-error.test.js

```diff
@@ -50,7 +50,7 @@ describe('generatePaymentExchangeRateError', () => {
         row: 1,
       },
       'payment exchange rate': {
-        value: 'abc',
+        value: null,
```

Comment:
Are we sure that the csv parser will turn this to `null` if the cell is empty? I haven't looked too much into it, but I could see the values for these things being `null`, `undefined` or `''` when parsed 🤷

Reply:
Just tested, it's parsing empty cells as `null`